service/src/main/java/bio/terra/workspace/app/controller/shared/PropertiesUtils.java

```diff
@@ -0,0 +1,31 @@
+package bio.terra.workspace.app.controller.shared;
+
+import bio.terra.workspace.common.utils.ControllerValidationUtils;
+import bio.terra.workspace.generated.model.ApiProperties;
+import bio.terra.workspace.generated.model.ApiProperty;
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/** Utils for properties that are key value pairs in WSM workspace and resources. */
+public class PropertiesUtils {
```

Comment:
why? this is different from the rest of model which is mostly pojo class that stores fields of an object. This one is just a helper to do conversion so i think this is just a utils.

Reply:
This is in controller directory. I think of conversion/data model stuff more related to models and controllers. It can be something Utils.java. Up to you.